src/pages/Staking/Staking.columns.tsx

```diff
@@ -53,3 +55,8 @@ export const formatStakingPeriod: CellParser = (val) => {
 
 export const formatFishAmountColumn: CellParser = (val) =>
   `${formatWeiAmount(val)} FISH`;
+
+export const TxHashColumn: HashColumnComponent = ({ value }) => {
+  const account = useSelector(accountSelector);
+  return value !== account ? <PrettyTx value={value} /> : <>-</>;
```

Comment:
I don't get it. Why are you comparing hash of transaction with user address?

Why can't we just always show the tx hash?